stable/prometheus-mongodb-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "prometheus-mongodb-exporter.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+    helm.sh/chart: {{ include "prometheus-mongodb-exporter.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+{{ toYaml .Values.annotations | indent 8 }}
+      labels:
+        app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          env:
+            - name: MONGODB_URL
+              value: {{ .Values.config.datasource }}
+              {{- if .Values.config.auth }}
+            - name: HTTP_AUTH
+              value: {{ .Values.config.auth }}
```

Comment:
This variable is missing from the values.yaml. Also, give an example of the format as it's unclear.